vm/src/stdlib/sys.rs

```diff
@@ -217,6 +217,16 @@ mod sys {
         Vec::new()
     }
 
+    #[pyattr]
+    fn orig_argv(vm: &VirtualMachine) -> Vec<PyObjectRef> {
+        vm.state
+            .settings
+            .orig_argv
```

Comment:
does `orig_argv` only used for this function? Then I think settings.orig_argv is not necessary.

```suggestion
        std::env::args().
```

Reply:
I agree with you. I checked the python doc and the `orig_argv` is not called anywhere else.
I'll remake this then.